mne/io/reference.py

```diff
@@ -266,6 +266,22 @@ def add_reference_channels(inst, ref_channels, copy=True):
     return inst
 
 
+_ref_dict = {
+    FIFF.FIFFV_MNE_CUSTOM_REF_ON: 'on',
+    FIFF.FIFFV_MNE_CUSTOM_REF_OFF: 'off',
+    FIFF.FIFFV_MNE_CUSTOM_REF_CSD: 'CSD',
+}
+
+
+def _check_can_reref(inst):
+    _validate_type(inst, (BaseRaw, BaseEpochs, Evoked), "Instance")
+    current_custom = inst.info['custom_ref_applied']
+    if current_custom not in (FIFF.FIFFV_MNE_CUSTOM_REF_ON,
+                              FIFF.FIFFV_MNE_CUSTOM_REF_OFF):
```

Comment:
I don't get this condition test.

Reply:
(This conditional could equivalently be written as `if current_custom == FIFFV_MNE_CUSTOM_REF_CSD: raise RuntimeError(...)` but it's less future compatible with other "custom ref"s we might eventually add)

Reply:
ok thanks for clarifying